CONTRIBUTING_DOCS.md

```diff
@@ -93,57 +91,77 @@ Let's say there's a single commit that makes changes to the `master` directory w
 
-Each orchestrator has a landing page that is targeted at people who are coming to see Calico for the first time. It's a transition from the "marketing" type material (why is Calico great) to some quick commands people can run to see it firsthand, and then funnels people off to the Usage section for more details.
+### Operations
 
-### Usage
+This section contains post-install, task-based content.  
 
-This section contains task-based information. All top-level titles in this section should start with a gerund. Each topic should include why you want to perform the task, a goal, and a set of steps you can follow to achieve it.
+### Networking
 
-Examples:
+This section contains task-based content for networking using the Calico CNI and Calico IPAM.
 
-- Configuring BGP peers
-- Enabling IP-in-IP in AWS
-- Troubleshooting Calico
-- Using calicoctl in a Kubernetes deployment
-- Configuring egress policy in Kubernetes
+### Security
 
-Do not include detailed description of components or tabulated
-configuration information in this section. This type of content should be located in the [Reference](#reference) section.
+This section contains task-based content for securing Calico components, workloads, and non-cluster hosts using Calico network policy.
 
 ### Reference
 
-These docs contain complete reference information for Calico. If there's a configuration option you're looking for, it goes here in one of the per-component references. Not every option has a "how to" guide, but has enough description. The caveats and considerations when enabling options should be listed here.
+This section contain reference content including APIs and Resources. Add configuration options in one of the per-component references, and list any caveats and considerations when enabling options.
 
 Examples:
 
 - Fully tabulated configuration options per-component
 - `calicoctl` help text
 - Calico API schema reference (policy, ip pool, etcd)
 
-
 ## Organizational changes
 
 ### Creating new pages
 
-- To create a top level splash page for a URL path, simply name the file `index.md`.
+- To create a top level splash page for a URL path, name the file `index.md`. 
+
+  If the index.md has child topics, add the following. The `description` field must be lowercase or you will get an error during the build. For example:
+
+  ```
+  ---
+  title: Install Calico
+  description: Install Calico on nodes and hosts for popular orchestrators, and install the calicoctl command line interface (CLI) tool. 
+  canonical_url: '/getting-started/index'
+  show_read_time: false
+  show_toc: false
+  ---
+
+  {{ page.description }}
 
+  {% capture content %}{% include index.html %}{% endcapture %}
+  {{ content | replace: "    ", "" }}
+  ```
```

Comment:
It looks a bit unmaintainable to include such detail here.  Why not instead recommend just to copy and tweak an existing index.md file in the same section?

Reply:
There is definitely a risk that this becomes a maintenance pain point as Neil says.  And I was thinking we also need to included instructions for how to do external links and how to do tabs.  I think it would be more contributor friendly to have these incantation explicit in this doc rather than ask people to go and find examples of them.  These are not quite a long magic incantations as the index pages, but there's probably not a lot of difference.  So maybe just put them all here explicitly and we live with the maintenance pain in order to give contributors an easier life?

Reply:
(That sounds fine; I just wanted to raise this as a possible concern, in case it had been missed.)

Reply:
fixed.